core-backend/common/src/lib.rs

```diff
@@ -109,4 +109,7 @@ pub trait Environment<E: Ext + IntoExtInfo + 'static>: Sized {
     /// Run setuped instance starting at `entry_point` - wasm export function name.
     /// - IMPORTANT: env is in inconsistent state after execution.
     fn execute(&mut self, entry_point: &str) -> Result<BackendReport, BackendError>;
+
+    /// Unset env ext and returns gas amount.
+    fn drop_env(&mut self) -> GasAmount;
```

Comment:
Why do we need this inside the common and general trait?

Reply:
Because, now we has needs to be able drop env before execution